repos/system_upgrade/el8toel9/actors/cloud/checkvalidgrubcfghybrid/libraries/checkvalidgrubcfghybrid.py

```diff
@@ -0,0 +1,27 @@
+from leapp import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import HybridImage
+
+
+def process():
+    hybrid_image = next(api.consume(HybridImage), None)
+
+    if hybrid_image:
+        reporting.create_report([
+            reporting.Title(
+                'Azure hybrid (BIOS/EFI) image detected. The GRUB configuration might be regenerated.'
+            ),
+            reporting.Summary(
+                'Leapp detected the system is running on Azure cloud, booted using BIOS In case '
+                'of such a hybrid image scenario and upgrading from older systems (i.e. RHEL 7) '
+                'it is possible that the system might end up with invalid GRUB configuration, '
+                'as `/boot/grub2/grub.cfg` might be overwritten by an old configuration from '
+                '`/boot/efi/EFI/redhat/grub.cfg`, which might cause the system to fail to boot. '
+
+                'Please ensure that the system is able to boot with both of these '
+                'configurations. If an invalid configuration is detected during upgrade, '
+                'it will be regenerated automatically using `grub2-mkconfig.`'
+            ),
+            reporting.Severity(reporting.Severity.HIGH),
+            reporting.Groups([reporting.Groups.PUBLIC_CLOUD]),
```

Comment:
A `reporting.Groups.BOOT` groups is missing from this list.

Reply:
Added! Thanks!